contrib/babelfishpg_tsql/src/tsqlIface.cpp

```diff
@@ -617,7 +617,7 @@ static std::string getProcNameFromExecParam(TSqlParser::Execute_parameterContext
 		TSqlParser::Execute_body_batchContext *exBodyBatchCtx = dynamic_cast<TSqlParser::Execute_body_batchContext *>(ctx->parent);
 		TSqlParser::IdContext *proc = nullptr;
 
-		if (exBodyCtx != nullptr)
+		if (exBodyCtx != nullptr && exBodyCtx->proc_var == nullptr)
```

Comment:
would you mind add a test case related to this change

Reply:
Added